aezeed/cipherseed.go

```diff
@@ -506,8 +505,13 @@ func (m *Mnemonic) Decipher(pass []byte) ([DecipheredCipherSeedSize]byte, error)
 	// Before we attempt to map the mnemonic back to the original
 	// ciphertext, we'll ensure that all the word are actually a part of
 	// the current default word list.
+	wordDict := make(map[string]bool, len(defaultWordList))
```

Comment:
Seems we can accomplish the same thing with a `map[string]struct{}` since the value isn’t read?

Reply:
yep, even better, thanks :+1: